packages/ui/tsconfig.json

```diff
@@ -9,7 +9,7 @@
 		"types": ["vitest/globals"],
 		"baseUrl": "src",
 		"paths": {
-			"@/*": ["./src/*"],
+			"@/*": ["./*"],
```

Comment:
Was also trying to fix import aliases. This got it working in the editor, but our builds were failing to resolve the aliases.